Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM nvidia/cuda
+
+# Install miniconda
+RUN apt-get update && apt-get install -y wget
+RUN MINICONDA="Miniconda2-latest-Linux-x86_64.sh" && \
+    wget --quiet https://repo.continuum.io/miniconda/$MINICONDA && \
+    bash $MINICONDA -b -p /miniconda && \
+    rm -f $MINICONDA
+ENV PATH /miniconda/bin:$PATH
+
+# Add omnia
+RUN conda config --add channels omnia
+
+# Install deepchem
+RUN conda install --yes deepchem
```

Comment:
Note this is a VERY old version of deepchem.

I can't even figure out the git hash which corresponds to this -- it has a "featurizer" package vs a "feat" package.  It is at least pre September 2015.  

I guess this just forces us to get on our conda package creation game, and create an alpha release.